packages/resourcehacker.vm/resourcehacker.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>resourcehacker.vm</id>
-    <version>5.1.8</version>
+    <version>5.2.7.20240413</version>
```

Comment:
Not a big fan of naming the package `reshack.portable.vm` as I think it's cleaner to have the tool name. I change it to `5.2.7.20240413` because I hit same error as https://github.com/mandiant/VM-Packages/pull/741#issuecomment-1823190837

Reply:
Since `resourcehacker.portable` is working too I suggest following nuspec file so that we keep same package name :
```xml
    <id>resourcehacker.vm</id>
    <version>5.2.7</version>
    <dependencies>
      <dependency id="common.vm" />
      <dependency id="resourcehacker.portable" version="[5.2.7]" />
    </dependencies>
```

Reply:
there are only two options to follow the documentation:
1. rename the package to `reshack.portable.vm`
2. change the package version to `0.0.0.DATE`

Using `resourcehacker.portable` as dependency, `resourcehacker.vm` as package name, and `5.2.7` as package version **does not follow the documentation**. Using this setting will cause that our bot updates the dependency version, but not the package version (as it is not a metapackage because the name of the dependency doesn't match the package name). We had have this issue with other packages that were not following the documentation (and we need to fix them): https://github.com/mandiant/VM-Packages/issues/985. So, if you want to keep `resourcehacker.vm` as the package name, you need to change the package version to `0.0.0.DATE`.

Reply:
As mentioned above, it is not possible to change it to `0.0.0.20240422`. By default the latest version is install based on test script and CI/CD (which is 5.1.8 from your repository `https://www.myget.org/F/vm-packages/api/v2`). If you can remove it from myget it will work but I don't see other solution.

Reply:
I have based my code on the example you provide in the comment `cygwin` : https://github.com/mandiant/VM-Packages/blob/main/packages/cygwin.vm/cygwin.vm.nuspec#L5
A change might be required there too

Reply:
Sorry I was not clear, I'll delete the previous version from myget after you do the change decreasing the version.

cygwin does not need to be changed. The dependency and the package have the exact same name, so it is considered to be a metapackage. The issue with your changes is that you want to keep two different names and in this case the package is not considered a metapackage and the dependency is updated without updating the package name. As mentioned already, there are only two options to follow the documentation:
1. rename the package to `reshack.portable.vm`
2. change the package version to `0.0.0.DATE`

Reply:
Oh, that's why, thanks for the explanation. I have update the package as request